models/Post.php

```diff
@@ -61,15 +61,15 @@ class Post extends Model
      * @var array
      */
     public static $allowedSortingOptions = [
```

Comment:
Just change this to a static method that uses Lang if you want to translate it. I don't want several component properties each defining their own available options.

Reply:
I got the "Constant expression contains invalid operations" error message. I will search a better solution.